commons/src/main/java/gov/cms/qpp/conversion/model/error/ErrorCode.java

```diff
@@ -203,14 +203,11 @@ private String subValues(Object... arguments) {
 		return new StrSubstitutor(valueSub, "`(", ")`").replace(getMessage());
 	}
 
-	private static Pattern getPattern() {
-		if (replacePattern == null) {
-			replacePattern = Pattern.compile(VARIABLE_MARKER);
-		}
-		return replacePattern;
-	}
-
 	public static ErrorCode getByCode(int code) {
 		return CODE_TO_VALUE.get(code);
 	}
+
+	private static final class VariableMarker {
+		private static final Pattern REPLACE_PATTERN = Pattern.compile("`\\(([^()]*)\\)`");
```

Comment:
Instead of creating a private inner class, why don't we just continue to use `VARIABLE_MARKER` on line 136 but have it be a `Pattern` instead of a `String`?  Does this static inner class not get initialized until it is referenced or something?

Reply:
The way java enums + static fields work, this is required.

Reply:
Ah, OK.  Then we should remove the unused `VARIABLE_MARKER` and `replacePattern` on line 136 and 137.